code/routing/FluentRootURLController.php

```diff
@@ -99,4 +99,22 @@ public function handleRequest(SS_HTTPRequest $request, DataModel $model = null)
 		$this->popCurrent();
 		return $result;
 	}
+    
+    /**
+     * 
+     * With Translatable installed, don't pre-append the locale to the homepage 
+     * URL.
+     * 
+     * @return string
+     * @see {@link Translatable::get_homepage_link()}.
+     */
+    public static function fluent_homepage_link($localeURL) {
+        $homepageLink = parent::get_homepage_link();
+        if(class_exists('Translatable')) {
```

Comment:
Added warm, fuzzy comment block.
Not sure what you mean by "Fluent homepage". Do you mean a homepage ala `/en/home`? That works AFAIKT..